tailcfg/tailcfg_test.go

```diff
@@ -639,30 +638,6 @@ func TestNetInfoFields(t *testing.T) {
 	}
 }
 
-type keyIn interface {
-	String() string
-	MarshalText() ([]byte, error)
-}
-
-func testKey(t *testing.T, prefix string, in keyIn, out encoding.TextUnmarshaler) {
```

Comment:
I think the tests use a testdata file, but it looks like maybe this is the code we use to generate that file. Should we maybe move it to a utils package someplace?

Reply:
I don't see a testdata file in this package. The existing tests mostly all have hand-rolled JSON strings.
This function was added in https://github.com/tailscale/tailscale/commit/88c305c8afa15dbee5a6cb8b98e5547cd19f4e7e to test the marshal/unmarshal roundtrip. Not sure how to repurpose it for generating test data.